src/ui/components/UrlExportSettings.tsx

```diff
@@ -71,9 +72,13 @@ export const UrlExportSettings = () => {
         event_type: settings.eventType,
         client_payload: {
           tokens: `${stringifyJson(tokensToExport, settings.urlJsonCompression)}`,
-          filename: `${settings.filename}${settings.extension}`
+          filename: `${settings.filename}${settings.extension}`,
+          message: `${message}`
```

Comment:
please rename all instances of message to `commitMessage`, it's better if we are precise. Who knows hat comes later.

Reply:
Done.